src/main/features/ccj/views/repayment-plan.njk

```diff
@@ -26,7 +26,8 @@
         <div class="form-group">
           <fieldset>
             <legend class="visually-hidden">{{ t('Date for first payment') }}</legend>
-            {{ dateInput('firstPaymentDate', form, hint = 'For example, 12 03 2017',
+            {{ dateInput('firstPaymentDate', form,
+            hint = t('For example, {{ nextMonth }}', {nextMonth : 'now' | addDays(30) | inputDate}),
```

Comment:
what's `inputDate` do vs `date`?

Reply:
`inputDate` formats it as "dd MM yyyy" (31 7 2018) whereas `date` formats it as "dd MMMM yyyy" (31 July 2018)